ClassHomework/src/Harry_Potter_Reader/WordCounter.java

```diff
@@ -0,0 +1,54 @@
+package Harry_Potter_Reader;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class WordCounter {
+	static HashMap<String, Integer> wordCount = new HashMap<String, Integer>();
+	
+	public static void main(String[] args) {
+		long start = System.currentTimeMillis();
+		try {
+			
+			File script = new File("harry_potter.txt");
+			Scanner s = new Scanner(script);
```

Comment:
just remember to put the file under the project, not the src or inside the package :) 👍 (already fixed during xblock)